common/queue/src/main/java/org/thingsboard/server/queue/kafka/TbKafkaAdmin.java

```diff
@@ -84,6 +86,12 @@ public void createTopicIfNotExists(String topic) {
 
     }
 
+    int getPartitionsForTopic(String topic) {
```

Comment:
A very "simple" implementation that violates basic OOP principles. You should add a parameter to the TbKafkaConsumerTemplate builder  (and other queue implementations as well).

Reply:
@ashvayka, the mentioned before `TB_QUEUE_KAFKA_` parameters are related only for Kafka implementations. As we are using the same config for request and response topics, the single responsible class for creating topics is TbKafkaAdmin. So I have no clear idea of how to apply this solution with better implementation at this moment.

The only thing I can do earlier in the builder stage is to maintain a separate config for the response topic based on the request topic config by overriding the partition parameter with exactly 1. Should I?